generic_relations/relations.py

```diff
@@ -42,6 +42,9 @@ def field_to_native(self, obj, field_name):
         Delegates to the `to_native` method of the serializer registered
         under obj.__class__
         """
+        if not getattr(obj, field_name):
```

Comment:
@Ian-Foote On second thought, maybe we should check to see if the attribute exists before dealing with the value, which might be normalized by the serializer to u'' or whatever.